Purchases.xcodeproj/project.pbxproj

```diff
@@ -1750,6 +1753,7 @@
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				MTL_ENABLE_DEBUG_INFO = NO;
 				SDKROOT = iphoneos;
+				SUPPORTS_MACCATALYST = YES;
```

Comment:
this is the vital change so that direct xcode integration works correctly on catalyst